cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1553,14 +1553,20 @@
   </property>
 
   <property>
+    <name>metadata.updates.kafka.broker.list</name>
+    <value></value>
```

Comment:
nit: is it a good idea to leave this empty? My intellij complains that xml cannot have tags with empty values. Having a default value will not hurt, no?